composer.json

```diff
@@ -31,15 +31,20 @@
   },
   "require": {
     "php": "^7.0",
+    "ext-openssl": "*",
```

Comment:
You should probably not require ext-openssl, but rather have a `function_exists` in the `enableSignature` method and throw an error if not. This way, you may still use it on a system that doesn't support openssl, and also allow an openssl polyfill (which effectively does not exist yet in PHP, but we never know 😄 ).